core/src/components/input/input.ionic.scss

```diff
@@ -1,10 +1,14 @@
 @import "./input";
 @import "./input.ionic.vars";
+@import "./input.ionic.outline.scss";
 
 // Ionic Input
 // --------------------------------------------------
 
-
+:host {
+  --border-width: 1px;
+  --border-color: #B8B8B8;
+}
```

Comment:
I went ahead and put these styles here instead of the outline-specific file since they'll be needed for the underline appearance as well.